scripts/swagger_update/app.py

```diff
@@ -29,6 +29,9 @@
         # storage package-2019-06 references privatelinks.json which is not listed in the input files
         "storage": { 
             "package-2019-06": ["./Microsoft.Storage/stable/2019-06-01/privatelinks.json"] 
+        },
+        "cosmos-db" : {
+            "package-2020-04" : ["./Microsoft.DocumentDB/stable/2019-08-01/cosmos-db.json"]
```

Comment:
wooo I did something useful!

Reply:
YES - very much YES!

Reply:
Seeing that made me happy as I was working on this 😄